homework-11/script.js

```diff
@@ -0,0 +1,113 @@
+//Bai 1:
+//cach 1:
```

Comment:
Không được dùng tiếng Việt
Chú ý dấu cách
```suggestion
// Bai 1:
// Cach 1:
```